tasks.py

```diff
@@ -1,64 +1,86 @@
 #!/usr/bin/env python3
 """
 A simple task manager.
-
 For the purposes of our exercise we store tasks in a file which we'll
 call tasks.csv (a CSV file). We'll talk more later about other ways to
 store state which are more robust.
 """
 from tempfile import NamedTemporaryFile
 import csv
 import os
+import sys
 
 
-DONE = " ✅"
+#DONE = " ✅"
+DONE = " \u2705"
 
 
-def list():
+def list(stdout=sys.stdout, filename="tasks.csv"):
     """
     List the current known tasks.
     """
-
-    with open("tasks.csv") as tasks_file:
-        reader = csv.reader(tasks_file)
-        for name, completed in reader:
-            print(f"{name}{DONE if completed else ''}")
-
-
-def create(name):
+    # HW TASK 1: check box ONLY if completed=="True"
+    try:
+        with open(filename) as tasks_file:
+            reader = csv.reader(tasks_file)
+            for name, completed in reader:
+                output = name 
+                if completed=="True":
+                    output = name + DONE
+                stdout.write(f"{output}\n")
+    except FileNotFoundError:
+        pass
+
+
+def create(name, filename="tasks.csv"):
     """
     Create a new task.
     """
-
-    with open("tasks.csv", "a") as tasks_file:
+    # added newline option to address compatibility for Windows
+    with open(filename, "a", newline='') as tasks_file:
         writer = csv.writer(tasks_file)
         writer.writerow([name, False])
 
 
-def complete():
+def complete(filename='tasks.csv',to_complete=None):
     """
     Mark an existing task as completed.
     """
 
     with (
-        open("tasks.csv") as tasks_file,
-        NamedTemporaryFile("w", delete=False) as new,
+        # added newline option to address compatibility for Windows
+        open(filename,newline='') as tasks_file, 
+        NamedTemporaryFile("w", delete=False,newline='') as new,
     ):
         reader = csv.reader(tasks_file)
         print("Current tasks:")
+        # HW TASK 2: index tasks to begin counter at 1
         for id, (name, completed) in enumerate(reader):
```

Comment:
Nice -- enumerate also takes a second argument, the start number, so you can also have fixed this via enumerate(reader, 1) rather than i + 1.